Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md

```diff
@@ -1,17 +1,20 @@
 <!-- HTML_DOC -->
 <p>Use the Microsoft Graph integration to connect to and interact with data on Microsoft Platforms. This integration was integrated and tested with Microsoft Graph v1.0.</p>
-<p> </p>
 <h2>Use Cases</h2>
-<p> </p>
 <ol>
 <li>Manage alerts</li>
 <li>Manage users</li>
 </ol>
-<p> </p>
+<h2>Required Permissions in the MS Graph Security App:</h2>
```

Comment:
please also mention the type of the permission: Delegated or Application
im pretty sure they are all Application but need to verify